### PR TITLE
Setup cart popover

### PR DESCRIPTION
**Pull request by mrand15**

### CODE REVIEW

File: src/components/Header/Cart/index.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          height={16}
+          className="text-main"
+        />
+        <span className="text-grey-dark">
+          &nbsp;Cart
+        </span>
+      </div>
+      <div
+        style={{ width: '256px' }}
+        className={classnames("header-cart-popover right-side absolute right-0 flex flex-column z4 rounded bxs-wrapper mt1 bg-white", {
```
Comment (dissimulate) [inline src/components/Header/Cart/index.js]: When you get beyond 4 or so classes it's best to create a new class and `@extend .class1, .class2...`, keeps the JSX tidy

File: src/styling/pages/checkout.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    flex: 2 1 0;
+    padding: 40px;
+    margin-right: 8px;
+    box-shadow: $box-shadow-light;
+  }
+
+  #checkout-summary {
+    background-color: white;
+    flex: 1 1 0;
+    padding: 40px 32px;
```
Comment (dissimulate) [inline src/styling/pages/checkout.scss]: We should be using rem/em over px most of the time since they're scalable, px isn't good for accessibility

Comment (mrand15) [reply src/styling/pages/checkout.scss]: I'm not a big fan honestly. Scaling everything up and down can break the layout in anything but the most simple design. It doesn't even do anything unless you change root level font-size.

Comment (mrand15) [reply src/styling/pages/checkout.scss]: I'd rather use media queries to adjust the layout

**Pull request merged**